commit e27654df5c688988fbab318d735b48f051a3d1f7
Author: Alexander Mette <mail@amette.eu>
Date:   Sun Nov 17 15:55:05 2013 +0100

    Fix #6357 - Don't remove useful signature files
    
    duplicity >=0.6.20 now correctly removes useless signature files and
    therefore the cache directory doesn't grow infinitely any more. Using
    --extra-clean removes all signatures except from the last chain and so
    renders older chains unusable, don't use it by default any more.
    See: https://bugs.launchpad.net/duplicity/+bug/519948

diff --git a/handlers/dup.in b/handlers/dup.in
index a67848f..1ce7710 100644
--- a/handlers/dup.in
+++ b/handlers/dup.in
@@ -191,7 +191,9 @@ else
 fi
 
 ### Cleanup options
-execstr_options="${execstr_options} --extra-clean"
+if [ "$duplicity_major" -le 0 -a "$duplicity_minor" -le 6 -a "$duplicity_sub" -le 19 ]; then
+   execstr_options="${execstr_options} --extra-clean"
+fi
 
 ### Temporary directory
 if [ -n "$tmpdir" ]; then
